cli/operator/node.go

```diff
@@ -122,6 +122,7 @@ var StartNodeCmd = &cobra.Command{
 		cfg.SSVOptions.Context = ctx
 		cfg.SSVOptions.DB = db
 		cfg.SSVOptions.BeaconNode = el
+		cfg.SSVOptions.Eth1Client = cl
```

Comment:
I know this was in the code beforehand, but this is funny, or some one was confused. 
EL - execution layer - ETH1 (geth)
CL - consensus layer - ETH2 (beacon node)